pkg/java/pom/parse.go

```diff
@@ -52,6 +52,7 @@ type parser struct {
 	localRepository    string
 	remoteRepositories []string
 	offline            bool
+	settings           settings
```

Comment:
we duplicate `localRepository`.
It might be better to just use serverSettings(`[]Server`).